wandb/sdk/internal/internal_api.py

```diff
@@ -376,6 +381,34 @@ def api_key(self) -> Optional[str]:
         default_key: Optional[str] = self.default_settings.get("api_key")
         return env_key or key or sagemaker_key or default_key
 
+    @property
+    def access_token(self) -> Optional[str]:
+        """
+        Retrieves an access token for authentication.
+
+        This function attempts to exchange an identity token for a temporary access token from the server, and save it
```

Comment:
My bad - I thought the formatting checks would do this for me.

Reply:
It might be nice to add more lint rules around docstrings though, either through one of the commit hooks or as a build step that would fail. Especially for folks like myself that are newer to the format

Reply:
Yeah, you're right, we should enable checks on line length. Human attention will still be necessary for quality, in particular about information and conciseness.

One challenge with docstrings is that they can't generally be auto-formatted, so we'd have to update all our current ones by hand or ignore files, but there are several active multi-thousand line files out there 🥲